cpp/src/parquet/statistics.cc

```diff
@@ -627,7 +640,12 @@ class TypedStatisticsImpl : public TypedStatistics<DType> {
   T min_;
   T max_;
   ::arrow::MemoryPool* pool_;
-  int64_t num_values_ = 0;  // # of non-null values.
+  // # of non-null values.
+  // num_values_ would be reliable when has_null_count_,
+  // if it's created from a page thrift statistics, and statistics
+  // doesn't have null_count, but page has null data, `num_values_`
+  // would be equal to value including nulls.
```

Comment:
This isn't very readable. Do you want to split this into two sentences?

Reply:
Tried it but maybe I'm not good for docs